src/main/java/Duke/ParseInfo.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class ParseInfo {
```

Comment:
I feel this class name would sound better if it is made to sound less imperative, e.g. `InfoParser`